### ISSUE TITLE
HZTSafe 1/2 Mile Solar Wireless Driveway Alarm

### ISSUE THREAD

**Issue by stutteringp0et** [merbanan/rtl_433_tests#419]
I haven't had any luck locating the manufacturer website, but this brand/model of driveway alarm is available everywhere - amazon, walmart, ebay - it's all over the place.  The model number is HB-T001Q3 and I have found its FCC application info.  Not only does the model number match, but the diagrams show the particular units I'm working with.

https://fccid.io/2AJP6HB-T001Q3/

In the attached sample files, I triggered the alarm twice.  I am fairly remote, and running rtl_433 to listen to my environment found no signals, so this test should be clean of interference.  The receiver unit has the ability to assign different tones to each sensor, so I assume that each sensor has a unique ID

[rtl_433.zip](https://github.com/merbanan/rtl_433_tests/files/8064635/rtl_433.zip)
.

**Comment by klohner** [comment]
@stutteringp0et - did you figure this out?  I took a look at the samples you'd attached and there's nothing useful in them.

Based on the [FCC ID page](https://fccid.io/2AJP6HB-T001Q3/) you'd referenced, these devices operate at 434.6 MHz but you seem to have recorded your samples at 433.92 MHz and I'd suspect the signal was not received.  If you're still interested, please provide some samples recorded at the correct frequency and I'll take a look.

**Comment by kramttocs** [comment]
Also interested in this. Does 433.92 not also capture 434.6?

Or is that not what this means?

"433.050 MHz to 434.790 MHz"

**Comment by zuckschwerdt** [comment]
No, 434.6M is out of band. It's a band of "sample rate"-width. E.g. 433.795 to 434.045 MHz at the default 433.92 "center frequency" and 250 kHz sample rate.

**Comment by zuckschwerdt** [comment]
Hard to imagine that a simple "doorbell"-style sender would use FHSS. But that would show as hard to capture, something more like a LoRA signal.

**Comment by kramttocs** [comment]
That sounds unfortunate

**Comment by klohner** [comment]
It's out of band by default, but it's simple enough just to specify the frequency you want to listen to.

Something this should record the signal for further analysis:

`rtl_433 -f 434.6M -s 1k -S all`

**Comment by kramttocs** [comment]
[rtl_433_driveway_1.zip](https://github.com/user-attachments/files/16823474/rtl_433_driveway_1.zip)

Thanks. Do these contain anything?

**Comment by zuckschwerdt** [comment]
No, there is no signal. Drop files on https://triq.org/pdv/ -- if you see just smooth color there is nothing.
